apps/settings/elements/firefox_accounts.html

```diff
@@ -29,14 +29,14 @@ <h2 data-l10n-id="fxa-confirm-account">Confirm Your Account</h2>
       </header>
       <ul>
         <li class="description">
-          <p data-l10n-id="fxa-verification-email-sent-msg" id="fxa-unverified-text">
```

Comment:
yes. The entity requires an attribute (email address), so it will not work without it, so there's no reason to keep it in HTML since it has to be defined from JS as l10n-id/l10n-args couple and it is in `apps/settings/js/firefox_accounts/panel.js#showUnverifiedPanel`